app/src/main/res/layout/continue_playing_fragment_test_activity.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:tools="http://schemas.android.com/tools"
-  android:id="@+id/continue_playing_fragment_placeholder"
+  android:id="@+id/recently_played_fragment_placeholder"
```

Comment:
The final should also be changed in this.

Reply:
Done